### DISCUSSION TITLE
After removing all elements from datagrid, the view is not rendering newly added rows

### DISCUSSION THREAD

**Discussion by KalinNikolov** [vmware/clarity#5595] [Q&A]
I'm using clarity v4.0.11 with angular v10.0.2.
I have two datagrid components and I'm adding elements from the first datagrid to the second. I'm also removing elements from the second one. So when the second one becomes empty, the view is not updated after adding new elements to it. Here is a stackblitz --> https://stackblitz.com/edit/datagrid-add-elements?file=src/app/app.component.ts

Steps are:
1. Select element from first datagrid and add it to the second one.
2. Select the newly added in the second datagrid and remove it.
3. Try again to add from first datagrid to the second.

Result: 
Element is added, but the view is not updated.

Expected result:
Element is added and view is updated.

Anyone experiencing the same issue?

Kalin Nikolov

**Comment by hippee-lee** [comment]
The stackblitz seems to have some issues, I'm not seeing Clarity styles applied or any rows of data in the first datagrid.
We have several starters that already configure the ClarityModule and css for a demo app here: https://stackblitz.com/@clr-team

Can update with a working, simplified POC for what you are trying?

**Reply by KalinNikolov** [reply]
@hippee-lee  thanks, for noting that, I've updated the stackblitz, so it is now easier to view the issue.

**Reply by KalinNikolov** [reply]
Sure, I've opened an issue here: https://github.com/vmware/clarity/issues/5605
